flake.nix

```diff
@@ -3,12 +3,16 @@
 
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    nur.url = "github:nix-community/nur";
     flake-utils.url = "github:numtide/flake-utils";
   };
 
-  outputs = { self, nixpkgs, flake-utils }:
+  outputs = { self, nixpkgs, nur, flake-utils }:
     flake-utils.lib.eachDefaultSystem (system: let
-
+      nurpkgs = import nur {
+          nurpkgs = nixpkgs.legacyPackages.x86_64-linux;
+          pkgs = nixpkgs.legacyPackages.x86_64-linux;
```

Comment:
This doesn't evaluate on `aarch64` systems, and the `cabal-docspec` in `nur` seems be [downloading a binary](https://github.com/nix-community/nur-combined/blob/1207aa33351d6fd3a68de49326431effedbf3e4a/repos/amesgen/pkgs/default.nix#L93).

Reply:
Can we build `cabal-docspec` as a normal Haskell package in our flake?

Reply:
That may be possible. Not sure when I'll have the time to play around with that but it seems the easiest path forward. Too bad it isn't on hackage.

Reply:
There is also this https://github.com/phadej/cabal-extras/issues/134